File: app/src/main/luaLibs/helper/DialogHelper.lua
local LuaHelper={}

---设置内容是否可选
---@param dialog Dialog 对话框实例
---@param selectable boolean 是否可选
function LuaHelper.setMessageIsSelectable(dialog,selectable)
  local textView = dialog.findViewById(android.R.id.message)
  textView.setTextIsSelectable(selectable)
end
return LuaHelper

File: app/src/main/java/io/gitee/jesse205/github/urlconverter/helper/DialogHelper.java
package io.gitee.jesse205.github.urlconverter.helper;

import android.app.Dialog;
import android.widget.TextView;

public class DialogHelper {
    public static void setMessageIsSelectable(Dialog dialog, boolean selectable) {
        TextView textView = dialog.findViewById(android.R.id.message);
        textView.setTextIsSelectable(selectable);
    }
}
